tests/ui/layout/rust-call-abi-not-a-tuple-ice-81974.stderr

```diff
@@ -6,7 +6,7 @@ LL | impl<A, B> FnOnce<A> for CachedFun<A, B>
    |
 note: required by a bound in `FnOnce`
   --> $SRC_DIR/core/src/ops/function.rs:LL:COL
-help: consider further restricting this bound
+help: consider further restricting this bound but it is an `unstable` trait
```

Comment:
nit: `unstable` is not a keyword, and "but" feels awkward here... maybe put it in parens and use "although" instead?

> consider further restricting this bound (although this is an unstable trait)

Reply:
maybe "although note that it is unstable" in the parentheses?

Reply:
I also think the "it" here is a bit unclear, on first reading I thought "it" referred to `T` and was then thrown off by `T` being an unstable trait

Perhaps the trait could be named or something like "the new bound" or "the bound added below" be used instead?

Reply:
the "it" is the trait, not really the bound

Reply:
sure, but if you're going to use "but" please still put it in parentheses or at least add a comma, since it's still a bit awkward grammatically

Reply:
Drive-by comment: perhaps it would be easier if the types were explicitly named? For example:

```
error[E0277]: functions with the "rust-call" ABI must take a single non-self tuple argument
  --> $DIR/rust-call-abi-not-a-tuple-ice-81974.rs:31:5
   |
LL |     extern "rust-call" fn call_once(mut self, a: A) -> Self::Output {
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Tuple` is not implemented for `A`
   |
help: consider adding the unstable trait `Tuple` to the bounds of `A`
   |
LL |     A: Eq + Hash + Clone + std::marker::Tuple,
   |                          ++++++++++++++++++++
```

Then the stability or non-stability can be added via the phrase "consider adding the trait `Tuple` to the bounds of `A`" / "consider adding the unstable trait `Tuple` to the bounds of A`".

Alternatively, it could be suggested via a separate note:

```
error[E0277]: functions with the "rust-call" ABI must take a single non-self tuple argument
  --> $DIR/rust-call-abi-not-a-tuple-ice-81974.rs:31:5
   |
LL |     extern "rust-call" fn call_once(mut self, a: A) -> Self::Output {
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `Tuple` is not implemented for `A`
   |
help: consider adding the trait `Tuple` to the bounds of `A`
   |
LL |     A: Eq + Hash + Clone + std::marker::Tuple,
   |                          ++++++++++++++++++++
note: the trait `Tuple` is unstable
```